generators/app/templates/src/utils/arrayUtils.js

```diff
@@ -9,5 +9,5 @@ export function stringArrayToObject(actionsArray, namespace = '') {
   if (actionsArray.some(actionName => !actionName || typeof actionName !== 'string')) {
     throw new Error('Action names must be strings and must not be empty');
   }
-  return Immutable(actionsArray).asObject(actionName => [actionName, `${namespace}:${actionName}`]);
+  return Immutable(actionsArray).asObject(actionName => [actionName, `${namespace}${actionName}`]);
```

Comment:
I'd keep the colon for readability, any particular reason for removing it?

Reply:
We can conditionally remove it if there is no namespace

Reply:
you are right! removing that colon was not a good decision